src/main/java/net/datafaker/providers/science/ScienceFaker.java

```diff
@@ -0,0 +1,35 @@
+package net.datafaker.providers.science;
+
+import net.datafaker.providers.base.BaseFaker;
+import net.datafaker.service.FakeValuesService;
+import net.datafaker.service.FakerContext;
+import net.datafaker.service.RandomService;
+
+import java.util.Locale;
+import java.util.Random;
+
+public class ScienceFaker extends BaseFaker implements ScienceProviders {
```

Comment:
I'm not sure we need this class right now
We could think about such extraction once there are at least 5 providers in this category

Reply:
Sure, moved under the base package